internal/staticupload/staticupload.go

```diff
@@ -24,8 +24,12 @@ import (
 	"github.com/aws/aws-sdk-go-v2/service/cloudfront"
 	cftypes "github.com/aws/aws-sdk-go-v2/service/cloudfront/types"
 	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/google/uuid"
 )
 
+// DefaultDistributionID is the default CloudFront distribution ID to use.
+const DefaultDistributionID = "E1H77EZTHC3NE4"
```

Comment:
```suggestion
```

nit: I would move this to internal/constants/constants.go. Now you have many packages importing `staticupload` just to access this constant.